src/nfhl/main.py

```diff
@@ -240,6 +240,22 @@ def _update_hazard_layer_symbology(gis):
     return result
 
 
+def _delete_existing_gdb_item(gis, gdb_item_name, module_logger):
+    module_logger.info("Searching for and deleting GDB item '%s' if needed...", gdb_item_name)
+
+    searches = gis.content.search(query=gdb_item_name, item_type="File Geodatabase")
+    if not searches:
+        module_logger.info("GDB item '%s' not found, proceeding", gdb_item_name)
+        return
+    if len(searches) > 1:
+        raise ValueError(f"Multiple GDBs found matching {gdb_item_name}")
+    gdb_item = searches[0]
```

Comment:
**question**: it seems a little strange that 

1. that there can be multiple gdb's with the same name. Are they in different folders?
2. that only the first return is deleted. wouldn't the same problem still exist?

Reply:
Good points, I had to go double check my logic.

1. I'm using the AGOL query language for the search, so I don't trust it won't find other similarly named items as well. This is just me trying to be safe.
2. Since the preceding lines either return for 0 or raise for >1, line 252 should only run if there's only one item found (unless I'm missing something obvious?)